<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_insertar">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Insertar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Actualizar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Elimnnar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_marca">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_modelo">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Marca</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Modelo</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_marca">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>91</width>
     <height>271</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Marca</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="table_model">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>101</width>
     <height>281</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Modelo</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="buscar_marca">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="buscar_modelo">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Buuscar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
